Makefile

```diff
@@ -37,11 +37,16 @@ report:
 
 # Run PostgreSQL
 localdb:
-	docker compose -f compose.dev.yml up
+	docker compose -f compose.dev.yml up -d
```

Comment:
You shouldn't really do this, since this is meant to be used for local development. To keep in foreground and be able to shut it down using ctrl + c